src/utils/zIndex.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Standardize the use of zIndex across the application
+ *
+ * Reference: https://www.notion.so/audiusproject/Client-Modal-Design-cc9d722f2aed4a0abf14fb44f97c246e
```

Comment:
I'd remove this reference since it's for modals and not exposed in notion. i think we should just get better about documentation inside code -- like let's specify around these two zIndexes why we have them. If you think some of the notes there are super useful, let's just copy the text into this doc-comment.

// Set to 1000 to account for nested modals inside, which take a higher z-index
EDIT_TRACK_MODAL = 1000